### PR TITLE
[MDS-6118] Fixed numbering of permit conditions when missing

### PR DESCRIPTION
**Pull request by simensma-fresh**
## Objective 

[MDS-6118](https://bcmines.atlassian.net/browse/MDS-6118)

- Fixed an issue where conditions without numbering extracted would have one generated.
- Fixed an issue that pops up sometimes where the status of the permit condition extraction task would be updated to "SUCCESS" before the conditions are saved to the DB

image

### CODE REVIEW

File: services/permits/docker-compose.yaml [hunk 1]
```diff
@@ -102,8 +102,8 @@ services:
         echo "Setting kibana_system password";
         until curl -s -X POST --cacert /certs/ca/ca.crt -u "${ELASTICSEARCH_USERNAME}:${ELASTICSEARCH_PASSWORD}" -H "Content-Type: application/json" https://elasticsearch:9200/_security/user/kibana_system/_password -d "{\"password\":\"${ELASTICSEARCH_PASSWORD}\"}" | grep -q "^{}"; do sleep 10; done;
         sleep 3;
-        echo "All done!";
-      '
+        echo "All done!"; tail -f /dev/null
```
Comment (simensma-fresh) [inline services/permits/docker-compose.yaml:105]: This should fix the `create_certs` container error that sometimes pops up

File: services/core-api/app/api/mines/permits/permit_conditions/models/permit_conditions.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 from app.api.utils.models_mixins import AuditMixin, Base, SoftDeleteMixin
 from app.extensions import db
 from marshmallow import fields, validate
-from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.dialects.postgresql import JSON, UUID
```
Comment (taraepp) [inline services/core-api/app/api/mines/permits/permit_conditions/models/permit_conditions.py]: JSON import added by accident? I don't see it being used

File: services/permits/.env-example [hunk 3]
```diff
@@ -11,9 +11,10 @@ PERMIT_SERVICE_ENDPOINT=http://localhost:8004
 PERMITS_CLIENT_ID=mds-core-api-internal-5194
 PERMITS_CLIENT_SECRET=
 ELASTICSEARCH_CA_CERT=
-ELASTICSEARCH_HOST=
-ELASTICSEARCH_USERNAME=
-ELASTICSEARCH_PASSWORD=
+ELASTICSEARCH_HOST=https://elasticsearch:9200
+ELASTICSEARCH_USERNAME=elastic
+ELASTICSEARCH_PASSWORD=elastic
+PERMITS_CLIENT_SECRET=
```
Comment (taraepp) [inline services/permits/.env-example]: heads up- PERMITS_CLIENT_SECRET is also on line 12

**Review by matbusby-fw — DISMISSED** [review]
Looks good to me! (recognizing the couple of things Tara has mentioned)

**Review by henryoforeh-dev — APPROVED** [review]

**Review by taraepp — APPROVED** [review]

**Review by matbusby-fw — APPROVED** [review]

**Pull request merged**